src/components/calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="cal-container">
+        <span className="output">0</span>
+        <span className="btn-container-one">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button type="button" className="bg-color">÷</button>
+        </span>
+        <span className="btn-container-two">
+          <button type="button">7</button>
+          <button type="button">8</button>
+          <button type="button">9</button>
+          <button type="button" className="bg-color">×</button>
+        </span>
+        <span className="btn-container-three">
+          <button type="button">4</button>
+          <button type="button">5</button>
+          <button type="button">6</button>
+          <button type="button" className="bg-color">-</button>
+        </span>
+        <span className="btn-container-four">
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+          <button type="button" className="bg-color">+</button>
+        </span>
+        <span className="btn-container-five">
+          <button type="button" className="zero">0</button>
+          <button type="button">.</button>
+          <button type="button" className="bg-color">=</button>
+        </span>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] As requested by the previous reviewer and in order to align with the project requirements, it's advisable to incorporate at least one child component within your Calculator react component. This not only enhances code organization but also contributes to better maintainability.

Reply:
@levy002  Sorry about that. I didn't push the changes.
Could you please check it again, I just want to be sure if my changes were correct.